frontend/projects/valtimo/layout/src/lib/components/widget-management/management-content/table/widget-management-table.component.scss

```
@@ -42,10 +41,17 @@
   &__path-selector {
     display: flex;
     flex-direction: column;
+    padding: 8px;
+    background-color: var(--cds-layer-01);
+
+    ::ng-deep .cds--text-input,
```

Stylelint (17.11.1)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)